projects/OG-Financial/src/main/java/com/opengamma/financial/analytics/isda/credit/CreditCurveDataKey.java

```diff
@@ -67,6 +68,19 @@
    */
   @PropertyDefinition
   private final RestructuringClause _restructuring;
+
+  /**
+   * The Credit Default Swap type. Defaults to Single Name
+   */
+  @PropertyDefinition
```

Comment:
Only the name and currency is needed. The Index uses those, the single name uses the snr and restructuring. The type defaults to Single name